tests/test_data.py

```diff
@@ -150,14 +143,14 @@ async def test_parse_new_report_event():
 
     new_report = await parse_new_report_event(cfg, tx_hash, 0.50)
 
+    cfg.endpoints.endpoints.remove(endpoint)
```

Comment:
Looks like an infura key is hardcoded on line 147/140, then removed here. Intentional?

Reply:
ah yeah, it's only added for the purposes of the test